lib/Resizable.jsx

```diff
@@ -91,16 +92,14 @@ export default class Resizable extends React.Component {
 
   render() {
     let p = this.props;
-    let className = p.className ?
```

Comment:
Yeah that's fine by me - I just like the class names to be there so that:

* We don't break backcompat, and
* There's an easy extension point.